src/Cool/DidYouKnow/testDidYouKnowModal.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Cool/DidYouKnow/testDidYouKnowModal.hpp"
+#include <fmt/chrono.h>
+#include <imgui.h>
+
+void did_you_know(Cool::DidYouKnowModal& _did_you_know)
```

Comment:
This function is not related to testing, it should be in the main didYouKnowModal file.